src/valid/analyzer.rs

```diff
@@ -689,13 +685,7 @@ impl FunctionInfo {
                 non_uniform_result: self.add_ref(expr),
                 requirements: UniformityRequirements::empty(),
             },
-            E::CallResult(function) => {
-                let info = other_functions
-                    .get(function.index())
-                    .ok_or(ExpressionError::CallToUndeclaredFunction(function))?;
```

Comment:
We can also remove `ExpressionError::CallToUndeclaredFunction` since this was the only place it was used.